spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -247,15 +279,15 @@ def get_channel_ids(self):
         return self.channel_ids.tolist()
 
     @staticmethod
-    def write_recording(recording, save_path, acquisition_name='ElectricalSeries', **nwbfile_kwargs):
+    def write_recording(recording, save_path, metadata={}):
```

Comment:
I prefer `metadata=None` and then having 

```
if metadata is None:
    metadata = dict()
```

I know it's a little extra code, but having a mutable as an input arg can lead to some weird behavior I'd rather avoid